site/src/components/Abbr/Abbr.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * @file A more sophisticated version of the native <abbr> element.
+ *
+ * Features:
+ * - Better type-safety (requiring you to include certain properties)
+ * - All built-in HTML styling is stripped away by default
+ * - Better integration with screen readers, with more options for controlling
+ *   how they read out initialisms.
+ */
+import { visuallyHidden } from "@mui/utils";
+import { type FC, type HTMLAttributes } from "react";
+
+// There isn't a dedicated HTMLAbbrElement type; have to go with base type
+type AbbrProps = Exclude<HTMLAttributes<HTMLElement>, "title" | "children"> & {
+  // Not calling this "title" to make it clear that it doesn't have the same
+  // issues as the native title attribute as far as screen reader support
```

Comment:
I don't know that this rename lends a lot of clarity, tbh - I might just use "title" so there's less cognitive overhead

Reply:
Yeah, I can see that. As I've been doing research, it seems that `title` is usually next to useless for screen readers, but reframing it like "the API is 99% the same as the native `<abbr>`, and we fixed the `title` prop" would make things more for other folks